admin-dev/themes/default/css/vendor/OverlayScrollbars.min.css

```diff
@@ -0,0 +1,13 @@
+/*!
```

Comment:
These files can be removed, you already add it in package.json

Reply:
I was able to install the dependancy and I made the change.
You can check it.